src/openforms/logging/tests/test_events.py

```diff
@@ -33,3 +44,61 @@ def test_submission_start(self):
 
             log = TimelineLogProxy.objects.last()
             self.assertEqual(_("Anonymous user"), log.fmt_user)
+
+    def test_submission_logic_evaluated(self):
+
+        json_logic_trigger = {
+            ">": [{"date": {"var": "birthdate"}}, {"date": "2022-06-20"}]
+        }
+
+        json_logic_trigger_2 = {"==": [{"var": "firstname"}, "bar"]}
+
+        actions = [
+            {
+                "component": "firstname",
+                "formStep": "",
+                "action": {
+                    "type": "property",
+                    "property": {"value": "disabled", "type": "bool"},
+                    "state": True,
+                },
+            }
+        ]
+
+        actions_2 = [
+            {
+                "component": "",
+                "formStep": "",
+                "action": {
+                    "type": "disable-next",
+                    "property": {"type": "", "value": ""},
+                    "value": "",
+                    "state": "",
+                },
+            }
+        ]
```

Comment:
I don't think we really do anything with the actions here in the test, so maybe don't specify them at all? When creating the rule, you can use `actions=[]` and it'll be allowed by the database :)